src/taco-cli/cli/run.ts

```diff
@@ -88,6 +91,12 @@ class Run extends commands.TacoCommandBase {
             if (!remoteConfig) {
                 throw errorHelper.get(TacoErrorCodes.CommandRemotePlatformNotKnown, platform);
             }
+            
+            // DeviceSync/LiveReload not compatible with remote
+            var deviceSync = commandData.options["livereload"] || commandData.options["devicesync"];
+            if (deviceSync) {
```

Comment:
Currently, we are not supporting livereload on remote scenarios.
After discussing it, we realized that as the current remote architecture stands, it wouldn't work anyway and would require significant architectural changes to make it work. So, we left it out of this scope of work.